ipvs/netlink.go

```diff
@@ -402,6 +402,14 @@ func (i *Handle) doGetServicesCmd(svc *Service) ([]*Service, error) {
 	return res, nil
 }
 
+// doCmdWithoutAttr a simple wrapper of netlink socket execute command
+func (i *Handle) doCmdWithoutAttr(cmd uint8) error {
+	req := newIPVSRequest(cmd)
+	req.Seq = atomic.AddUint32(&i.seq, 1)
+	_, err := execute(i.sock, req, 0)
```

Comment:
Instead of ignoring the return data, can you pls change `doCmdWithoutAttr` to return the data as well ?
The caller of this function can choose to ignore it if not useful.

Reply:
Done. PATL. Thanks!